examples/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "mocha": true
```

Comment:
I can do that if you prefer. Thought this made the most sense w/r/t getting the mocha environment setting out of the top-level `.eslintrc` without adding too many sub-`.eslintrc`s.

Your call.